smoke-test/src/test/java/org/opennms/smoketest/minion/OverlappingIPTest.java

```diff
@@ -0,0 +1,199 @@
+import org.opennms.netmgt.provision.persist.requisition.RequisitionInterface;
+import org.opennms.netmgt.provision.persist.requisition.RequisitionNode;
+import org.opennms.netmgt.snmp.SnmpObjId;
+import org.opennms.netmgt.snmp.SnmpTrapBuilder;
+import org.opennms.netmgt.snmp.SnmpUtils;
+import org.opennms.smoketest.NullTestEnvironment;
+import org.opennms.smoketest.OpenNMSSeleniumTestCase;
+import org.opennms.smoketest.utils.DaoUtils;
+import org.opennms.smoketest.utils.HibernateDaoFactory;
+import org.opennms.smoketest.utils.RestClient;
+import org.opennms.test.system.api.NewTestEnvironment.ContainerAlias;
+import org.opennms.test.system.api.TestEnvironment;
+import org.opennms.test.system.api.TestEnvironmentBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetSocketAddress;
+import java.util.Date;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+public class OverlappingIPTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OverlappingIPTest.class);
+
+    private static final String HOST = "1.2.3.4";
+
+    private static TestEnvironment environment;
+
+    private HibernateDaoFactory daoFactory;
+
+    @ClassRule
+    public static final TestEnvironment getTestEnvironment() {
+        if (!OpenNMSSeleniumTestCase.isDockerEnabled()) {
+            return new NullTestEnvironment();
+        }
+        try {
+            final TestEnvironmentBuilder builder = TestEnvironment.builder().all();
+            builder.withOpenNMSEnvironment()
+                   .addFile(SyslogTest.class.getResource("/eventconf.xml"), "etc/eventconf.xml")
+                   .addFile(SyslogTest.class.getResource("/events/Cisco.syslog.events.xml"), "etc/events/Cisco.syslog.events.xml")
+                   .addFile(SyslogTest.class.getResource("/syslogd-configuration.xml"), "etc/syslogd-configuration.xml")
+                   .addFile(SyslogTest.class.getResource("/syslog/Cisco.syslog.xml"), "etc/syslog/Cisco.syslog.xml");
+            OpenNMSSeleniumTestCase.configureTestEnvironment(builder);
+            return environment = builder.build();
+
+        } catch (final Throwable t) {
+            throw new RuntimeException(t);
+        }
+    }
+
+    @Before
+    public void checkForDocker() {
+        Assume.assumeTrue(OpenNMSSeleniumTestCase.isDockerEnabled());
+    }
+
+    @Before
+    public void setup() {
+        // Connect to the postgresql container
+        final InetSocketAddress pgsql = environment.getServiceAddress(ContainerAlias.POSTGRES, 5432);
+        this.daoFactory = new HibernateDaoFactory(pgsql);
+    }
+
+    @Test
+    public void test() throws Exception {
```

Comment:
Can we move this to the `SyslogTest` class instead? Most of the setup / methods are the same.

This also covers the functionality tested by `SyslogTest#canReceiveSyslogMessages`, so we can safely remove that one too.